client/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import AddTaskForm from './components/forms/AddTaskForm';
+import CurrentTasks from './components/forms/CurrentTasksForm';
+import CompletedTasks from './components/CompletedTasks';
+import RemoteSubmitButton from './components/remoteSubmitButton/RemoteSubmitButton';
+
+class App extends Component {
+  state = {
+    tasks: []
+  }
+
+  addTask = (task) => {
+    task.id = this.state.tasks.length;
+    let tasks = [...this.state.tasks, task];
```

Comment:
is there any reason for `let`?

Reply:
it's not OK to pass id like you did
usually id comes as a field of received task from server

Reply:
by the way, is this method `addTask` in use? it seems not